src/components/Cluster/NewCluster/ReleaseSelector/ReleaseSelector.tsx

```diff
@@ -24,6 +24,8 @@ import ReleaseRow from './ReleaseRow';
 interface IReleaseSelector {
   selectRelease(releaseVersion: string): void;
   selectedRelease: string;
+  collapsible?: boolean;
+  autoSelectLatest?: boolean;
```

Comment:
Added support for having the ReleaseSelector component non-collapsible (hide the drop-down caret icon and always have the component open), and for not selecting the latest version by default.